doc/tutorials/dataset_basics.rst

```diff
@@ -0,0 +1,600 @@
+
+    >>> print(ds)
+    (0008, 0005) Specific Character Set              CS: 'ISO_IR 100'
+    (0008, 0008) Image Type                          CS: ['ORIGINAL', 'PRIMARY', 'AXIAL']
+    (0008, 0012) Instance Creation Date              DA: '20040119'
+    (0008, 0013) Instance Creation Time              TM: '072731'
+    (0008, 0014) Instance Creator UID                UI: 1.3.6.1.4.1.5962.3
+    (0008, 0016) SOP Class UID                       UI: CT Image Storage
+    ...
+    (0010, 1002)  Other Patient IDs Sequence   2 item(s) ----
+        (0010, 0020) Patient ID                          LO: 'ABCD1234'
+        (0010, 0022) Type of Patient ID                  CS: 'TEXT'
+        ---------
+        (0010, 0020) Patient ID                          LO: '1234ABCD'
+        (0010, 0022) Type of Patient ID                  CS: 'TEXT'
+        ---------
+    ...
+    (0043, 104e) [Duration of X-ray on]              FL: 10.60060977935791
+    (7fe0, 0010) Pixel Data                          OW: Array of 32768 elements
+    (fffc, fffc) Data Set Trailing Padding           OB: Array of 126 elements
+
+The print output shows a list of the :dcm:`data elements
+<part05/chapter_7.html#sect_7.1>` (or *elements* for short) present in the
+dataset, one element per line. The format of each line is:
+
+* **(0008, 0005)**: The element's :dcm:`tag<part05/chapter_7.html#sect_7.1.1>`,
+  as (group number, element number) in hexadecimal
+* **Specific Character Set**: the element's name, if known
+* **CS**: The element's :dcm:`Value Representation<part05/sect_6.2.html>` (VR),
+  if known
+* **'ISO_IR_100'**: the element's stored value
+
+Elements
+--------
+
+There are three categories of elements:
+
+* **Standard elements** such as (0008,0016) *SOP Class UID*. These elements
+  are registered in the official DICOM Standard, have an even group
+  number and are unique at each level of the dataset.
+* **Repeating group elements** such as (60xx,3000) *Overlay Data* (not found
+  in this dataset). :dcm:`Repeating group<part05/sect_7.6.html>` elements are
+  also registered in the official DICOM Standard, however they have a group
+  number defined over a range rather than a fixed value.
+  For example, there may be multiple *Overlay Data* elements at a given level
+  of the dataset as long as each has its own unique group number; ``0x6000``,
+  ``0x6002``, ``0x6004``, or any even value up to ``0x601E``.
+* **Private elements** such as (0043,104E) *[Duration of X-ray on]*.
+  :dcm:`Private elements<part05/sect_7.8.html>` have an odd group number,
+  aren't registered in the official DICOM Standard, and are instead created
+  privately, usually by a manufacturer. In general, unless the manufacturer
+  publishes the details of their private elements, the element name and VR
+  aren't known. However, in this case the details have been made public and
+  we know the element name is *Duration of X-ray on* with a VR of **FL**.
```

Comment:
Maybe add that the meaning depends on the private creator tag (and that the example tag has that meaning only if the private creator is `GEMS_PARM_01`).